items/test_models.py

```diff
@@ -0,0 +1,46 @@
+from django.test import TestCase
+from .models import Item, Group
+from django.contrib.auth.models import User
+from django.utils import timezone
+from django.db import IntegrityError
+
+
+class ItemModelTest(TestCase):
+
+  def setUp(self):
+    super(ItemModelTest, self).setUp()
+    user = User.objects.create_user('user', 'u@u', 123)
+    group = Group(creator=user)
+    group.save()
```

Comment:
so basically you can create group w/o items (blank) ? it's not good, there is should be some validation on back-end